_posts/2017-09-18-impl-future-for-rust.md

```diff
@@ -0,0 +1,312 @@
+    <td>Dip your toes in with parsing and syntax sugar.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-compiler-middle</b></td>
+    <td>Implement features that involve typechecking.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-compiler-traits</b></td>
+    <td>Want generic associated types? You know what to do.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-compiler-incr</b></td>
+    <td>Finish incremental compilation; receive undying love.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-compiler-nll</b></td>
+    <td>Delve into the bowels of borrowck to slay the beast: NLL!</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-compiler-const</b></td>
+    <td>Const generics. Enough said.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+</table>
+
+### Libs team
+
+<table>
+<tr>
+    <td><b>WG-libs-blitz</b></td>
+    <td>Help finish off the Blitz before all the issues are gone!</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-libs-cookbook</b></td>
+    <td>Work on bite-sized examples to get folks cooking with Rust.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-libs-guidelines</b></td>
+    <td>Take the wisdom from the Blitz and pass it on.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-libs-simd</b></td>
+    <td>We want stable SIMD, and we need your help.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-libs-openssl</b></td>
+    <td>Want better docs for openssl? So do we.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-libs-rand</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+</table>
+
+### Docs team
+
+<table>
+<tr>
+    <td><b>WG-docs-rustdoc</b></td>
+    <td>Help make docs beautiful for everyone!</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-docs-rustdoc2</b></td>
+    <td>Get in on a bottom-up revamp of rustdoc!</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-docs-rbe</b></td>
+    <td>Teach others Rust in the browser.</td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+</table>
+
+### Dev tools team
+
+<table>
+<tr>
+    <td><b>WG-dev-tools-rls</b></td>
+    <td></td>
+    <td><a href="">Learn more</a></td>
+    <td><a href="">Chat</a></td>
+</tr>
+<tr>
+    <td><b>WG-dev-tools-vscode</b></td>
```

Comment:
Can we add WG-dev-tools-clients please? "Implement new clients for the RLS - Atom, Sublime Text, Visual Studio, etc."